packaging/dbscripts/upgrade/pre_upgrade/0000_config.sql

```diff
@@ -1354,6 +1390,38 @@ select fn_db_update_config_value('ServerCPUList',
         || '3:IBM z13s, z13:sie,model_z13-base:z13-base:s390x; '
         || '4:IBM z14:sie,model_z14-base:z14-base:s390x;',
     '4.7');
+select fn_db_update_config_value('ServerCPUList',
```

Comment:
This is not needed, you are adding ServerCPUList using `fn_db_add_config_value`. Only if you need to update ServerCPUList for 4.8 cluster level after 4.8 CL was already introduced, you would need to use `fn_db_update_config_value `